.size-snapshot.json

```diff
@@ -41,24 +41,5 @@
     "bundled": 2533,
     "minified": 1319,
     "gzipped": 541
-  },
-  "immer.js": {
-    "bundled": 789,
-    "minified": 376,
-    "gzipped": 214,
-    "treeshaked": {
-      "rollup": {
-        "code": 42,
-        "import_statements": 42
-      },
-      "webpack": {
-        "code": 1092
-      }
-    }
-  },
-  "immer.cjs.js": {
-    "bundled": 1228,
-    "minified": 676,
-    "gzipped": 307
```

Comment:
oops, I shouldn't have merged master... sorry. can you deal with it? we don't need `.size-snapshot.json` diff in this PR.

Reply:
@dai-shi 
Or you want me to rebase the `revert the change` commit with the original one?

Reply:
I mean, we can just delete L44-L62 here. Commits are not so important as I will squash them anyway.

Reply:
@dai-shi 
But I already commit it, and L44-L62 is already gone.

Reply:
@LuckyPigeon My apologies. It's the other way around. We need to put back deleted L44-62...

Reply:
@dai-shi 
Roger that & updated. Please check.